src/modules/tests/tests.cat-adi-part2.reducer.ts

```diff
@@ -34,7 +33,7 @@ export function testsCatADIPart2Reducer(
       accompaniment: accompanimentReducer,
       vehicleDetails: vehicleDetailsCatADIPart2Reducer,
       testData: testDataCatADI2Reducer,
-      passCompletion: passCompletionReducer,
+      passCompletion: () => null,
```

Comment:
Is this a temporary thing?

Reply:
No not really. It has to be null for ADI pt2 tests

Reply:
As it's common it can't be removed. Can we use the nullReducer instead.

Reply:
src/shared/classes/null.reducer.ts

Reply:
using `nullReducer` now